pycm/pycm_obj.py

```diff
@@ -656,24 +656,40 @@ def relabel(self, mapping):
         self.FP = self.class_stat["FP"]
         self.FN = self.class_stat["FN"]
         __class_stat_init__(self)
-    def weighted_average(self, item, weights=None):
+
+    def weighted_average(self, param, weight=None, none_omit=False):
         """
-        Calculate Weighted average of input parameter.
+        Calculate weighted average of input parameter.
```

Comment:
It should be written as follows:

"Calculate the weighted average of the input parameter"